backend/src/database.cpp

```diff
@@ -74,8 +95,8 @@ namespace database
             {
                 const char *error_msg = sqlite3_errmsg(db_);
                 fprintf(stderr, "SQLite error: %s\n", error_msg);
```

Comment:
**question**: Do we want this to throw an exception if below it aborts the code?

Reply:
Sure!

Reply:
Also, what other places need better error checking that throws errors instead of just fprintf errors?